mwapi/errors.py

```diff
@@ -36,3 +48,38 @@ class LoginError(RuntimeError):
     @classmethod
     def from_doc(cls, doc):
         return cls(doc.get('result'))
+
+
+class RequestError(requests.exceptions.RequestException):
```

Comment:
I do agree that it doesn't let us switch away, but in this case we're inheriting from requests exceptions so I don't see how that's any different.